src/commands/aksKaito/aksKaito.ts

```diff
@@ -17,12 +19,33 @@ export default async function aksKaito(_context: IActionContext, target: unknown
         return;
     }
 
+    if (!kubectl.available) {
```

Comment:
There's a check for kubectl below. Would the kubectl state have changed?

Reply:
This is just a check to make sure kubectl api via k8s extension is available. When you say below is it about creating temp file for kubeconfig? @ReinierCC

Reply:
Removed the duplicate lines Thanks @ReinierCC